### PR TITLE
Added some documentation when upgrading to 4.1.3 with ElasticSearch 7.10.2

### PR DESCRIPTION
**Pull request by bertrandgorge**
## Summary by CodeRabbit

- **New Features**
	- Introduced compatibility with ElasticSearch 7.10.2, including new configuration options for enhanced security.
- **Documentation**
	- Updated documentation to guide through the setup of Elasticsearch credentials.
- **Bug Fixes**
	- Addressed XSS vulnerabilities and improved security patches.
- **Refactor**
	- Adjustments made for ElasticStore users due to breaking changes in the update.

### CODE REVIEW

**Review by krabina — APPROVED** [review]

File: docs/releasenotes/RELEASE-NOTES-4.1.3.md [hunk 1]
```diff
@@ -71,3 +74,27 @@ This is only for those that have installed SMW via Git.
...
+                                  'user' => $elastic_user,
+                                  'pass' => $elastic_password ] ];
+
+$smwgElasticsearchCredentials = [
+    'host' => $elasticSearchHost,
+    'port' => 9200,
+    'scheme' => 'http',
+    'user' => $elastic_user,
+    'pass' => $elastic_password,
+];
```
Comment (marijnvanwezel) [inline docs/releasenotes/RELEASE-NOTES-4.1.3.md:97]:
```suggestion
$smwgElasticsearchCredentials = [
    'user' => $elastic_user,
    'pass' => $elastic_password,
];
```

File: docs/releasenotes/RELEASE-NOTES-4.1.3.md [hunk 2]
```diff
@@ -71,3 +74,27 @@ This is only for those that have installed SMW via Git.
...
+* Step 1: Upgrade your version of Elasticsearch to 7.10.2 (if you are using docker that should work without any other action than changing the version number)
+* Step 2: Make sure your version of the elasticsearch package is right, you might want to add it to the composer.local.json: `COMPOSER=composer.local.json composer require elasticsearch/elasticsearch "7.17.2"`
+* Step 3: Add $smwgElasticsearchCredentials after $smwgElasticsearchEndpoints:
+
+```
+$smwgElasticsearchEndpoints = [ [ 'host' => $elasticSearchHost,
+                                  'port' => 9200,
+                                  'scheme' => 'http',
+                                  'user' => $elastic_user,
+                                  'pass' => $elastic_password ] ];
```
Comment (marijnvanwezel) [inline docs/releasenotes/RELEASE-NOTES-4.1.3.md:89]:
```suggestion
$smwgElasticsearchEndpoints = [ 
    [
        'host' => $elasticSearchHost,
        'port' => 9200,
        'scheme' => 'http'
    ]
];
```

File: src/Elastic/docs/config.md [hunk 3]
```diff
@@ -10,13 +10,23 @@ Accessing an Elasticsearch cluster from within Semantic MediaWiki requires to al
...
+
+For SMW 4.1.3 and Elasticsearch 7.10.2 you also need to provide :
+```
+$smwgElasticsearchCredentials = [
+    'host' => '192.168.1.126',
+    'port' => 9200,
+    'scheme' => 'http',
+    'user' => $elastic_user,
+    'pass' => $elastic_password,
+];
```
Comment (marijnvanwezel) [inline src/Elastic/docs/config.md:28]:
```suggestion
$smwgElasticsearchCredentials = [
    'user' => $elastic_user,
    'pass' => $elastic_password,
];
```

**Review by marijnvanwezel — CHANGES_REQUESTED** [review]

**Comment by krabina** [comment]
Wikidoku added: https://www.semantic-mediawiki.org/w/index.php?title=Help%3AElasticStore&type=revision&diff=88859&oldid=80974

**Pull request merged**